azext_iot/central/params.py

```diff
@@ -96,6 +96,13 @@ def load_central_arguments(self, _):
             " Learn more at https://aka.ms/iotcentraldtmi.",
         )
 
+    with self.argument_context("iot central device-template list") as context:
+        context.argument(
+            "compact",
+            options_list=["--compact", "-c"],
+            help="Shot templates in compact mode",
```

Comment:
shot or short? Also update with what is shown in compact mode.

Reply:
typo. "show"